diff --git a/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m b/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m
index 7aa9b2a..1e0d853 100644
--- a/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m
+++ b/node_modules/react-native/React/Views/ScrollView/RCTScrollView.m
@@ -328,21 +328,21 @@ - (void)_keyboardWillChangeFrame:(NSNotification *)notification
   self.firstResponderFocus = CGRectNull;
 
   CGFloat contentDiff = 0;
-  if ([[UIApplication sharedApplication] sendAction:@selector(reactUpdateResponderOffsetForScrollView:)
-                                                 to:nil
-                                               from:self
-                                           forEvent:nil]) {
-    // Inner text field focused
-    CGFloat focusEnd = CGRectGetMaxY(self.firstResponderFocus);
-    BOOL didFocusExternalTextField = focusEnd == INFINITY;
-    if (!didFocusExternalTextField && focusEnd > endFrame.origin.y) {
-      // Text field active region is below visible area with keyboard - update diff to bring into view
-      contentDiff = endFrame.origin.y - focusEnd;
-    }
-  } else if (endFrame.origin.y <= beginFrame.origin.y) {
-    // Keyboard opened for other reason
-    contentDiff = endFrame.origin.y - beginFrame.origin.y;
-  }
+  // if ([[UIApplication sharedApplication] sendAction:@selector(reactUpdateResponderOffsetForScrollView:)
+  //                                                to:nil
+  //                                              from:self
+  //                                          forEvent:nil]) {
+  //   // Inner text field focused
+  //   CGFloat focusEnd = CGRectGetMaxY(self.firstResponderFocus);
+  //   BOOL didFocusExternalTextField = focusEnd == INFINITY;
+  //   if (!didFocusExternalTextField && focusEnd > endFrame.origin.y) {
+  //     // Text field active region is below visible area with keyboard - update diff to bring into view
+  //     contentDiff = endFrame.origin.y - focusEnd;
+  //   }
+  // } else if (endFrame.origin.y <= beginFrame.origin.y) {
+  //   // Keyboard opened for other reason
+  //   contentDiff = endFrame.origin.y - beginFrame.origin.y;
+  // }
   if (self.inverted) {
     newContentOffset.y += contentDiff;
   } else {
